apps/frontend/core-dapp/src/features/history/HistoryTransaction.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useMemo } from 'react'
+import { Flex, Icon, spacingIncrement } from 'prepo-ui'
+import styled from 'styled-components'
+import { observer } from 'mobx-react-lite'
+import Link from 'next/link'
+import { NETWORKS } from 'prepo-constants'
+import { HistoryTransaction as HistoryTransactionEntity } from './history.types'
+import { buttonColors as transactionTypeColors } from './History'
+import { numberFormatter } from '../../utils/numberFormatter'
+import PositionLabel from '../position/PositionLabel'
+import { getFullDateShortenMonthFromSeconds } from '../../utils/date-utils'
+import { useRootStore } from '../../context/RootStoreProvider'
+import { PositionName } from '../position/Position'
+
+const { toUsd } = numberFormatter
+
+const Label = styled.p`
+  color: ${({ theme }): string => theme.color.neutral1};
+  margin: 0;
+`
+
+const EventLink = styled.a<{ $event: string }>`
+  align-items: center;
+  background: ${({ $event, theme }): string =>
+    theme.color[transactionTypeColors[$event].backgroundColor]};
+  border-radius: ${spacingIncrement(8)};
+  color: ${({ $event, theme }): string => theme.color[transactionTypeColors[$event].color]};
+  display: flex;
+  padding: ${spacingIncrement(2)} ${spacingIncrement(8)};
+`
```

Comment:
ohh also add white-space: nowrap to this

```suggestion
const EventLink = styled.a<{ $event: string }>`
  align-items: center;
  background: ${({ $event, theme }): string =>
    theme.color[transactionTypeColors[$event].backgroundColor]};
  border-radius: ${spacingIncrement(8)};
  color: ${({ $event, theme }): string => theme.color[transactionTypeColors[$event].color]};
  display: flex;
  padding: ${spacingIncrement(2)} ${spacingIncrement(8)};
  white-space: nowrap;
`
```